ui/app/styles/components/modal.scss

```diff
@@ -86,6 +86,7 @@
 }
 
 .modal-card-body {
+  background-color: $white;
```

Comment:
every PR, I swear I find I miss one more style on the modals. It's difficult because we override the modal class in places so when looking at one modal it may have the right padding or background color, but an older modal without over-riding those classes won't. Anyways, rant done.